packages/docmanager-extension/src/index.tsx

```diff
@@ -54,7 +54,7 @@ import { ISignal, Signal } from '@lumino/signaling';
 import { Widget } from '@lumino/widgets';
 import * as React from 'react';
 import { recentsManagerPlugin } from './recents';
-
+import { DEFAULT_AUTOSAVE_INTERVAL } from '../../../notebook/constants';
```

Comment:
Import like will not work. It should import from `@jupyterlab/notebook/lib/constants` instead, although `lib` imports are best avoided.

Reply:
okay noted will change the 5 seconds to 10 seconds then